packages/sane_backends.rb

```diff
@@ -3,33 +3,34 @@
 class Sane_backends < Package
   description 'Scanner Access Now Easy - Backends'
   homepage 'http://www.sane-project.org/'
-  version '1.0.27'
+  version '1.0.32'
   compatibility 'all'
-  source_url 'https://gitlab.com/sane-project/backends/-/archive/RELEASE_1_0_27/backends-RELEASE_1_0_27.tar.bz2'
-  source_sha256 '6a4034a9d29255bb40cf5a19f3d85dd7ba4fe6452eb9b1ae4385bc8175493ec4'
+  source_url 'https://gitlab.com/sane-project/backends/uploads/104f09c07d35519cc8e72e604f11643f/sane-backends-1.0.32.tar.gz'
```

Comment:
`source_url "https://gitlab.com/sane-project/backends/uploads/104f09c07d35519cc8e72e604f11643f/sane-backends-#{version}.tar.gz"`?

Reply:
See the hash in the URL? That doesn't seem like a stable scriptable URL.

Reply:
How about `source_url "https://gitlab.com/sane-project/backends/-/archive/#{version}/backends-#{version}.tar.bz2"`?

Reply:
That works!